docs/the-branching-workflow.md

```diff
@@ -1,41 +1,8 @@
 # The Branching Workflow
 
-![The Branching Workflow](images/the-branching-workflow.png)
+Only two branches are maintained at any one time; these are `master` and one for [the current ownCloud release].
+Any change to the documentation is made in a feature branch, based off of `master`.
+Once the feature branch's PR is approved and merged, the PR is backported to the branch for the **current** ownCloud release.
```

Comment:
Normally a real feature would not be backported - it would e.g. be part of 10.4 release and that would branch off the end of master at the time of release.

Changes for patches-bugfixes will need to be backported (e.g. things released in 10.3.1 10.3.2 need their doc "fixes" backported to 10.3)

Reply:
> Normally a real feature would not be backported - it would e.g. be part of 10.4 release and that would branch off the end of master at the time of release.

Agreed. Poor wording on my part. Will revise.

Reply:
> Changes for patches-bugfixes will need to be backported (e.g. things released in 10.3.1 10.3.2 need their doc "fixes" backported to 10.3)

Will update the text to reflect this.

Reply:
@phil-davis; thoughts on the revised wording?